static/usage/v8/picker/theming/css-properties/angular/example_component_css.md

```diff
@@ -0,0 +1,36 @@
+```css
+ion-picker {
+  --highlight-background: #383a3e;
+  --highlight-border-radius: 50px;
+
+  --fade-background-rgb: 220, 221, 231;
```

Comment:
This looks a bit odd in dark mode: 
![image](https://github.com/ionic-team/ionic-docs/assets/2721089/0b54b07a-7480-473e-9ed8-cc7edfbd6fd8). You could change this to a color token instead maybe? I know we could do `--ion-background-color-rgb` but that kind of ruins the effect of the demo.

Reply:
I think this will make a big visual improvement:
```css
ion-picker {
   --highlight-background: var(--ion-color-dark);
  --highlight-border-radius: 50px;

  --fade-background-rgb: var(--ion-color-light-rgb);
}
```

|Light|Dark|
|---|---|
|![CleanShot 2023-12-13 at 17 42 28](https://github.com/ionic-team/ionic-docs/assets/13732623/10f30be5-6f69-476e-8ea6-cee0df3c6456)|![CleanShot 2023-12-13 at 17 42 07](https://github.com/ionic-team/ionic-docs/assets/13732623/7db3bba5-be98-46fe-b5bb-4d872cd93799)|

Reply:
works for me!